client/src/app/index.jsx

```diff
@@ -1,25 +1,45 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
+import { useInjectReducer } from 'redux-injectors';
+import { selectLoggedIn } from '../auth/selectors';
+import { sliceKey, reducer } from '../auth/slice';
+
 import { BrowserRouter } from 'react-router-dom';
 import { Switch, Route } from 'react-router-dom';
+import Account from './containers/Account';
 import Alert from './containers/Alert';
 import Home from './containers/Home';
 import Login from './containers/Login';
 import Navbar from './containers/Navbar';
+import NewGameForm from './containers/NewGameForm';
 import PrivateRoute from './containers/PrivateRoute';
 import Register from './containers/Register';
+import Sidebar from './components/Sidebar';
 import '../styles.css';
 
 export const App = () => {
+  useInjectReducer({ key: sliceKey, reducer: reducer });
+  const loggedIn = useSelector(selectLoggedIn);
+
   return (
     <BrowserRouter>
-      <div className='relative bg-gray-200 h-screen'>
-        <Navbar />
-        <Alert />
-        <Switch>
-          <Route exact path='/' component={Register} />
-          <Route exact path='/login' component={Login} />
-          <PrivateRoute exact path='/home' component={Home} />
-        </Switch>
+      <Navbar />
+      <div className='flex flex-row'>
+        {loggedIn && (
+          <div className='w-12 bg-blue-700'>
```

Comment:
Why wrap the `Sidebar` in a `div`?

Reply:
The `div` is handling the division of width between `Sidebar` and the rest, so `Sidebar` can just be full width. This `div` could be pulled into `Sidebar`, though :shrug: